app/models/event.rb

```diff
@@ -1,5 +1,6 @@
 class Event < ApplicationRecord
   EVENT_NAMES = [
+    NA = 'n/a'.freeze,
```

Comment:
Required because we have a database level null constraint. Happy to remove this later once we've moved off of the old version of events - if that happens.